packages/suite-data/.gitignore

```diff
@@ -1,2 +1,4 @@
 files/browser-detection
-files/message-system
\ No newline at end of file
+files/message-system
+/files/guide
```

Comment:
If #3815 gets merged before this we'll have a conflict that won't be detected by git.

Commenting here to remind myself before merging.

Reply:
Addressed in https://github.com/trezor/trezor-suite/pull/3811/commits/e3e932ef9108f38c92828a81bbf90d83b340fc21